lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -42,9 +78,19 @@ class LighthouseReportViewer {
      */
     this._reportIsFromGist = false;
 
-    this._addEventListeners();
-    this._loadFromDeepLink();
-    this._listenForMessages();
+    this._reportIsFromPSI = false;
+
+    const params = new URLSearchParams(location.search);
+    const url = params.get('url');
+    if (url) {
+      const categoriesCsv = params.get('categories');
```

Comment:
instead of one instance of query param `categories`, would it be better to have one `category` for each one? like the PSI api does.

Reply:
yeah let's do that way with the array param.

Reply:
done